spec/features/admin/admin_edits_order_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.feature do
+  before do
+    admin = create(:user, role: 1)
+    login_stub(admin)
+  end
+
+  context 'visit admin dashboard' do
+    context 'admin' do
+      scenario 'change order status from ordered to cancelled' do
+        create(:order_with_items, status: 0)
+        visit admin_dashboard_path
+
+        click_on 'cancel'
+
+        expect(Order.cancelled.count).to eq 1
```

Comment:
Can you throw an expectation of 0 before the cancel? I'm partial explicitly showing what a link does.